cmd/gossamer/config.go

```diff
@@ -157,20 +156,7 @@ func createInitConfig(ctx *cli.Context) (*dot.Config, error) {
 		return nil, err
 	}
 
-	// set global configuration values
-	err = setDotGlobalConfig(ctx, tomlCfg, &cfg.Global)
-	if err != nil {
-		logger.Errorf("failed to set global node configuration: %s", err)
-		return nil, err
-	}
-
-	if !cfg.Global.Pruning.IsValid() {
-		return nil, fmt.Errorf("--%s must be either %s or %s", PruningFlag.Name, pruner.Full, pruner.Archive)
-	}
-
-	if cfg.Global.RetainBlocks < dev.DefaultRetainBlocks {
-		return nil, fmt.Errorf("--%s cannot be less than %d", RetainBlockNumberFlag.Name, dev.DefaultRetainBlocks)
-	}
```

Comment:
why is the pruning settings removed here? pruning settings need to be set upon initialization

Reply:
Ah, I did not realize that!